### PR TITLE
Update settings description

### PR DESCRIPTION
**Pull request by karbobc**
Update description from `login` to `admin` in admin page.

`Read-Only Config`
image

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -545,7 +545,7 @@ make_config! {
 
         /// Seconds between admin requests |> Number of seconds, on average, between admin requests from the same IP address before rate limiting kicks in
         admin_ratelimit_seconds:       u64, false, def, 300;
-        /// Max burst size for login requests |> Allow a burst of requests of up to this size, while maintaining the average indicated by `admin_ratelimit_seconds`
+        /// Max burst size for admin requests |> Allow a burst of requests of up to this size, while maintaining the average indicated by `admin_ratelimit_seconds`
```
Comment (BlackDex) [inline src/config.rs:548]: Then, it actually should be `admin login` instead. Because now it implies on all admin requests, which is not the case.

**Review by BlackDex — CHANGES_REQUESTED** [review]

File: src/config.rs (hunk 1 above)
Comment (BlackDex) [reply src/config.rs:548]: the latter

Comment (BlackDex) [reply src/config.rs:548]: Also, this probably needs to be changed in the template: https://github.com/dani-garcia/vaultwarden/blob/7a7673103fb180098f18abe77b75ba085710b559/.env.template#L305

Comment (karbobc) [reply src/config.rs:548]: I'll update it at the same time.

Comment (karbobc) [reply src/config.rs:548]: > Also, this probably needs to be changed in the template:
> 
> https://github.com/dani-garcia/vaultwarden/blob/7a7673103fb180098f18abe77b75ba085710b559/.env.template#L305

This doesn't seem to need an update.

Comment (BlackDex) [reply src/config.rs]: Why not? It says `admin`, and not `admin login`. Would be best to keep both items consistent not?

Comment (karbobc) [reply src/config.rs]: You're right, I will update it later.

**Pull request merged**